Documentation/README.md

```diff
@@ -1,58 +1,62 @@
 Documents Index
 ===============
 
+Intro to .NET Core
+==================
+
+.NET Core is a self-contained .NET runtime and framework that implements ECMA 335. It can be (and has been) ported to multiple architectures and platforms. It support a variety of installation options, having no specific deployment requirements itself.
+
 Learn about .NET Core
 ====================
 
-- [Brief Intro to .NET Core](https://github.com/dotnet/coreclr/blob/master/Documentation/dotnetcore-intro.md)
 - [[WIP] Official .NET Core Docs](http://dotnet.readthedocs.org)
 
 Get .NET Core
 =============
 
-- [Get .NET Core DNX SDK on Windows](https://github.com/dotnet/coreclr/blob/master/Documentation/get-dotnetcore-dnx-windows.md)
-- [Get .NET Core DNX SDK on OS X](https://github.com/dotnet/coreclr/blob/master/Documentation/get-dotnetcore-dnx-osx.md)
-- [Get .NET Core DNX SDK on Linux](https://github.com/dotnet/coreclr/blob/master/Documentation/get-dotnetcore-dnx-linux.md)
-- [Get .NET Core (Raw) on Windows](https://github.com/dotnet/coreclr/blob/master/Documentation/get-dotnetcore-windows.md)
+- [Get .NET Core DNX SDK on Windows](https://github.com/dotnet/coreclr/blob/master/Documentation/install/get-dotnetcore-dnx-windows.md)
+- [Get .NET Core DNX SDK on OS X](https://github.com/dotnet/coreclr/blob/master/Documentation/install/get-dotnetcore-dnx-osx.md)
+- [Get .NET Core DNX SDK on Linux](https://github.com/dotnet/coreclr/blob/master/Documentation/install/get-dotnetcore-dnx-linux.md)
+- [Get .NET Core (Raw) on Windows](https://github.com/dotnet/coreclr/blob/master/Documentation/install/get-dotnetcore-windows.md)
 
 Project Docs
 ============
 
-- [Developer Guide](developer-guide.md)
-- [Project priorities](https://github.com/dotnet/coreclr/blob/master/Documentation/project-priorities.md)
-- [Contributing to CoreFX](contributing.md)
-- [Contributing to .NET Core](https://github.com/dotnet/coreclr/blob/master/Documentation/contributing.md)
-- [Contributing Workflow](https://github.com/dotnet/coreclr/blob/master/Documentation/contributing-workflow.md)
-- [Issue Guide](issue-guide.md)
-- [Branching Guide](branching-guide.md)
-- [API Review Process](api-review-process.md)
-- [Strong Name Signing](strong-name-signing.md)
-- [Open Source Signing](oss-signing.md)
-- [Repo Organization](repo-organization.md)
+- [Developer Guide](project-docs/developer-guide.md)
+- [Project priorities](https://github.com/dotnet/coreclr/blob/master/Documentation/project-docs/project-priorities.md)
+- [Contributing to CoreFX](project-docs/contributing.md)
```

Comment:
The ".NET Core Guidelines" link in that file is broken (commenting here since I couldn't comment in the file directly).

Btw. should we merge this file with the CONTRIBUTING.md file in the root given that it is very short and basically just points to the coreclr repo?